docs/guidance/example.md

```diff
@@ -4,32 +4,63 @@ The [OC4IDS schema](../reference/index.md) sets out the structure and format of
 
 This worked example is a fully completed OC4IDS JSON file representing a fictional, completed infrastructure project to upgrade a motorway in the UK.
 
-The full OC4IDS JSON file for the worked example is available to download [here](../../../../_static/example.json) and this page contains excerpts from the example JSON file, showing how key sections of the schema should be populated.
+The full OC4IDS JSON file for the worked example is available to download [here](../../../../_static/example.json).
+
+This page contains excerpts from the example JSON file, showing how key sections of the schema should be populated.
 
 ## Overview
 
 An OC4IDS document is made up of a number of sections. These include:
 
-* **Project metadata** - contextual information about the project e.g. title, description, location and status.
-* **Budget** - information about the projected costs or allocated budget for this project.
-* **Parties** - information about the organizations and other participants involved in this project.
-* **Documents** - documents and documentation relating to this project.
-* **Contracting processes** - information about related contracting processes for different aspects of the project.
-* **Completion** - information on the final scope, duration and costs for the project.
+* **Package metadata** - a container for data on multiple projects with important metadata about the publication
+* **Project information** - for each project including:
+  * **Project metadata** - contextual information about each project such as title, description, location and status.
+  * **Budget** - information about the projected costs or allocated budget for each project.
+  * **Parties** - information about the organizations and other participants involved in each project.
+  * **Documents** - documents and documentation relating to each project.
+  * **Contracting processes** - information about related contracting processes for different aspects of the project.
+  * **Completion** - information on the final scope, duration and costs for the project.
+
+## Sections
 
-The full JSON file for the example project looks like this:
+### Package metadata
+This section provides metadata about the data, including the schema version, publisher, data license and publication policy, and acts as a container for data on multiple projects in a `projects` array.
 
 ```eval_rst
 
 .. jsoninclude:: ../examples/example.json
-   :jsonpointer:  /projects/0
-```   
+   :jsonpointer:
+   :expand: publisher
+   :title: package
 
-## Sections
+```
+#### License
+
+The `license` field should contain a link to an appropriate data license. Further information about licensing can be found in our [OCDS guidance](https://standard.open-contracting.org/latest/en/implementation/licensing/).
```

Comment:
an appropriate data license > the license that applies to the data in the package.

our [OCDS guidance] > the [OCDS licensing guidance]